gatsby-browser.js

```diff
@@ -24,3 +25,48 @@ export const wrapRootElement = ({ element }) => {
     </BeaconProvider>
   );
 };
+
+const addScript = url => {
+  const script = document.createElement('script');
+  script.src = url;
+  script.async = true;
+  script.type = 'text/javascript';
+  script.charSet = 'utf-8';
+  script.onload = () => console.log('loaded: %s', url);
+  document.body.appendChild(script);
+};
+
+export const onRouteUpdate = ({ location, prevLocation }) => {
```

Comment:
@manangouhari  Please add a comment within the `onRouteUpdate` function to elaborate on the logical breakdown of the code.

Reply:
Done, take a look now.